packages/babel-preset-mc-app/index.js

```diff
@@ -43,8 +43,9 @@ module.exports = function getBabePresetConfigForMcApp() {
           // the targeted browsers. This is safer than `usage` which performs
           // static code analysis to determine what's required.
           // This is probably a fine default to help trim down bundles when
-          // end-users inevitably import '@babel/polyfill'.
+          // end-users inevitably import 'core-js/stable'.
           useBuiltIns: 'entry',
+          corejs: 3,
```

Comment:
Otherwise you rightfully get spammed with this warning

> WARNING: We noticed you're using the `useBuiltIns` option without declaring a core-js version. Currently, we assume version 2.x when no version is passed. Since this default version will likely change in future versions of Babel, we recommend explicitly setting the core-js version you are using via the `corejs` option.

> You should also be sure that the version you pass to the `corejs` option matches the version specified in your `package.json`'s `dependencies` section. If it doesn't, you need to run one of the following commands:

>  npm install --save core-js@2    npm install --save core-js@3
>  yarn add core-js@2              yarn add core-js@3